src/js/components/FieldMessage/FieldMessage.js

```diff
@@ -0,0 +1,41 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+import { getClassName } from '../../utils/helpers'
+
+class FieldMessage extends PureComponent {
+  render () {
+    const { children, isError, className, mods, style } = this.props
+
+    // TODO: Can we adjust this classStructure to be a modifier instead of replacement?
+    //  - FormGroup-message & FormGroup-feedback are almost identical except for color
```

Comment:
I think we kept these separate in case there were _both_ : a `.FormGroup-message` that is instructional and always shows, vs a `.FormGroup-feedback` which provides feedback based on form validation, and thus should be styled independently of the message. I suppose we could replace the instructional text with the feedback text.. and supply styles based on the `.is-invalid` class of the `FormGroup` as you suggest. Thoughts @williamdingwall?

Reply:
That's a good thought.  I wouldn't see both the message and error showing at the same time, I guess I thought them as replacing when needed.  But I could be wrong, either way I suppose.

Reply:
That's a good thought.  I wouldn't see both the message and error showing at the same time, I guess I thought them as replacing when needed.  But I could be wrong, either way I suppose.

Reply:
> I wouldn't see both the message and error showing at the same time, I guess I thought them as replacing when needed.

Does this get complicated to save and swap messages vs feedback for each field? For example, if message text is replaced by error text and then needs to go back to the message text once the field is valid? Alternately we could keep the separate elements (`-message` and `-feedback`) and hide and show each based on parent state.

> Can we adjust this classStructure to be a modifier instead of replacement?
Not sure what you mean by this.

Reply:
Nothing super complicated either way.   I think it's more of a design opinion on how you guys think it should operate.  Do you want to see the information message at all times even if an error is created or should the error temporarily replace the information message when triggered?

So if error and message are essentially the same field wouldn't it make more sense to make the class structure something like `className="FormGroup-message FormGroup-message--error"`

Reply:
Ah, I see what you mean now. Yes, using a modifier does make more sense.